### ISSUE TITLE
Using pabot?

### ISSUE THREAD

**Issue by patrijuh** [robotframework/MavenPlugin#62]
We are using RF maven plugin in our project and we have quite a lot of java code in our maven project. Also a lot of SUT configuration profiles exists in POM. As our tests runs get longer and longer, running our tests in parallel would be great. Would it be possible to use pabot with RF maven plugin? Any plans about supporting something like this in the future?

**Comment by Hi-Fi** [comment]
I have used CI to make the parallelization on the projects. Tests executed according tags, and CI then running rebot after all of the sets have completed.

Not sure about pabot possibilities directly, so would need checking.

**Comment by patrijuh** [comment]
Yes. We already do run tests in parallel in our CD pipes (tests executed by tags) but using pabot would be much better solution. Tests are executed on different nodes so using of rebot to generate reports is complicated.

**Comment by Hi-Fi** [comment]
I think the CI/CD approach of parallelization is still better and more generic, but pabot possibility can be looked at some point.

**Comment by tusharkr18** [comment]
Hello,
Is this enhancement available now?
